view/src/components/searchPage.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+import MemberList from './memberList';
+
+const SearchPage = (props) => {
+  const [input, setInput] = useState('');
+  const [memberListDefault, setMemberListDefault] = useState();
+  const [memberList, setMemberList] = useState();
+
+  const fetchData = async () => {
+    return await fetch('https://us-central1-swe-utd-portal.cloudfunctions.net/api/memberList')
```

Comment:
this should use Axios to keep consistency?

Reply:
I couldn't find any easy way to convert it to axios from the async function without getting some errors. For now, I can merge it and then look for a way to improve the consistency in my free time.